### ISSUE TITLE
[Legitimate Site Blocked] pfp-pit-brawl-production.s3.amazonaws.com

### ISSUE THREAD

**Issue by mintona87** [MetaMask/eth-phishing-detect#39087]
https://pfp-pit-brawl-production.s3.amazonaws.com/DecimalistWebGlbuild/index.html

Hello,

This is a legitimate site for our Unity WebGL browser-based game called PFP Pit Brawl. Please let us know what you need in order to remove the malicious flag. 

Thank you!

**Comment by AlexHerman1** [comment]
sorry, there is nothing hosted at this domain, are you saying this is a production URL for a game?

**Comment by mintona87** [comment]
Correct. It is a browser game that is stored in an AWS s3 bucket and then served via that link.

**Comment by AlexHerman1** [comment]
scheduled for unblocking!
